Source/WTF/wtf/Assertions.h

```diff
@@ -706,14 +706,14 @@ static constexpr bool unreachableForValue = false;
 #define RELEASE_LOG_FAULT(channel, ...) SUPPRESS_UNCOUNTED_LOCAL os_log_fault(LOG_CHANNEL(channel).osLogChannel, __VA_ARGS__)
 #define RELEASE_LOG_INFO(channel, ...) SUPPRESS_UNCOUNTED_LOCAL os_log_info(LOG_CHANNEL(channel).osLogChannel, __VA_ARGS__)
 #define RELEASE_LOG_DEBUG(channel, ...) SUPPRESS_UNCOUNTED_LOCAL os_log_debug(LOG_CHANNEL(channel).osLogChannel, __VA_ARGS__)
-#define RELEASE_LOG_WITH_LEVEL(channel, logLevel, ...) do { \
+#define RELEASE_LOG_WITH_LEVEL(channel, logLevel, fmt, ...) do { \
     if (LOG_CHANNEL(channel).level >= (logLevel)) \
-        SUPPRESS_UNCOUNTED_LOCAL os_log(LOG_CHANNEL(channel).osLogChannel, __VA_ARGS__); \
+        SUPPRESS_UNCOUNTED_LOCAL os_log(LOG_CHANNEL(channel).osLogChannel, fmt, __VAL_OPT__(, SAFE_PRINTF_TYPE(__VA_ARGS__))); \
```

Comment:
Note that it's `__VA_OPT__`:

 ```suggestion
        SUPPRESS_UNCOUNTED_LOCAL os_log(LOG_CHANNEL(channel).osLogChannel, fmt, __VA_OPT__(, SAFE_PRINTF_TYPE(__VA_ARGS__))); \
```

Please change the patch to use `__VA_OPT__` everywhere.